src/react-rails-ujs.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
```

Comment:
So I did some tests, obviously it would fail without `react-hot-loader` (I don't know what I was thinking). But `module.hot` is defined if Webpack is in Hot mode, which doesn't necessarily means that `react-hot-loader` is there and loaded. However, Webpack executes any `require` even if the condition is false, so you can't include the file even with `if (false) { AppContainer = require('react-hot-loader').AppContainer; }`.
Therefore, the only way it could work is with two files: one without `react-hot-reload` and one without. Would you find this solution acceptable?